tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -47,7 +47,11 @@ sub run {
     $variables{HANA_ACCOUNT} = get_required_var('QESAPDEPLOY_HANA_ACCOUNT');
     $variables{HANA_CONTAINER} = get_required_var('QESAPDEPLOY_HANA_CONTAINER');
     if (get_var('QESAPDEPLOY_HANA_TOKEN')) {
-        $variables{HANA_TOKEN} = get_required_var('QESAPDEPLOY_HANA_TOKEN');
+        $variables{HANA_TOKEN} = qesap_az_create_sas_token(storage => get_required_var('QESAPDEPLOY_HANA_ACCOUNT'),
+            container => get_required_var('QESAPDEPLOY_HANA_CONTAINER'),
```

Comment:
This one  for `container` is wrong. `QESAPDEPLOY_HANA_CONTAINER` is now used for two stuff:
- the request of the token: expected format is `CONTAINER_NAME`
- the value for `az_container_name` in qe-sap-deployment hana_media.yaml file: `CONTAINER_NAME/AND/ALL/SUBFOLDERS/WHERE/THE/INSTALLER/IS`